### ISSUE TITLE
Adding react-onClickOutside triggers error in create-react-class

### ISSUE THREAD

**Issue by jsonnull** [Pomax/react-onclickoutside#177]
I added react-onclickoutside to my build today and managed to trigger an error in create-react-class.

The error looks like:

```
Uncaught ReferenceError: React is not defined
    at Object.2../factory (app.js:7852)
    at s (app.js:7110)
    at e (app.js:7110)
    at app.js:7110
    at webpackJsonp../node_modules/create-react-class/create-react-class.js.i (app.js:7091)
    at Object../node_modules/create-react-class/create-react-class.js (app.js:7110)
```

I run a fairly stock babel+webpack setup, so I imagine others will run into this issue, which is why I'm documenting it here.

Looks like the React devs are aware of this and a fix is on the way:
https://github.com/facebook/react/issues/9689

**Comment by Pomax** [comment]
that's an interesting bug, thanks for the cross-reference!

**Comment by gaearon** [comment]
This should be fixed with `create-react-class@15.6.0-rc.0`. Please verify (I plan to publish the final `15.6.0` version of it tomorrow).

**Comment by Andarist** [comment]
Closing due to inactivity.
